CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Fixed
+- [#231][PR231] Fixed Marathon cluster code
```

Comment:
Still missing a link at the bottom of the page like:

`[PR231]: https://github.com/gambol99/go-marathon/pull/231`